src/Nodes.jl

```diff
@@ -4,26 +4,30 @@
 # Node.jl: helper function to access interface to Camera nodes
 
 # Utility functions
-function available(nodeName)
+function available(node)
   pbAvailable = Ref(bool8_t(false))
-  spinNodeIsAvailable(nodeName[], pbAvailable)
+  try
+    spinNodeIsAvailable(node.hNode[], pbAvailable)
+  catch err
+    error("Node handle invalid for $(node.name)\n$err")
+  end
   return (pbAvailable[] == 1)
 end
 
-function readable(nodeName)
-  if available(nodeName)
+function readable(node)
+  if available(node)
    pbReadable = Ref(bool8_t(false))
-   spinNodeIsReadable(nodeName[], pbReadable)
+   spinNodeIsReadable(node.hNode[], pbReadable)
    return (pbReadable[] == 1)
  else
    return false
  end
 end
 
-function writable(nodeName)
-  if available(nodeName)
+function writable(node)
+  if available(node)
    pbWriteable = Ref(bool8_t(false))
-   spinNodeIsReadable(nodeName[], pbWriteable)
+   spinNodeIsWritable(node.hNode[], pbWriteable)
```

Comment:
Just highlighting that there was a typo here. `spinNodeIsReadable` was being used in `writable()`